### PR TITLE
ThreadPoolTaskRunner: Make graceful shutdown logs less scary.

### PR DESCRIPTION
**Pull request by gianm**
- It's okay to suppress InterruptedException during graceful shutdown, as
  tasks may use it to accelerate their own shutdown.
- It's okay to ignore return statuses during graceful shutdown (which may
  be FAILED!) because it actually doesn't matter what they are.

### CODE REVIEW

File: indexing-service/src/main/java/io/druid/indexing/overlord/ThreadPoolTaskRunner.java [hunk 1]
```diff
@@ -159,8 +162,10 @@ public void stop()
...
               TimeUnit.MILLISECONDS
           );
+
+          // Ignore status, it doesn't matter for graceful shutdowns.
           log.info(
-              "Graceful shutdown of task[%s] finished in %,dms with status[%s].",
+              "Graceful shutdown of task[%s] finished in %,dms.",
               task.getId(),
               System.currentTimeMillis() - start,
               taskStatus.getStatusCode()
```
Comment (rasahner) [inline indexing-service/src/main/java/io/druid/indexing/overlord/ThreadPoolTaskRunner.java]: did you mean for this now-unused argument to stay here?

Comment (gianm) [inline indexing-service/src/main/java/io/druid/indexing/overlord/ThreadPoolTaskRunner.java]: whoops, good catch! will remove it.

**Comment by gianm** [comment]
https://travis-ci.org/druid-io/druid/builds/115348852

```
Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 6.356 sec <<< FAILURE! - in io.druid.indexing.worker.WorkerTaskMonitorTest
testRestartCleansOldStatus(io.druid.indexing.worker.WorkerTaskMonitorTest)  Time elapsed: 1.213 sec  <<< FAILURE!
java.lang.AssertionError: expected:<FAILED> but was:<RUNNING>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at io.druid.indexing.worker.WorkerTaskMonitorTest.testRestartCleansOldStatus(WorkerTaskMonitorTest.java:338)
```

**Comment by fjy** [comment]
:+1:

**Comment by nishantmonu51** [comment]
:+1:

**Pull request merged**